Sessions changes (#87)
Qiskit/documentation: docs/run/run-jobs-in-session.mdx

<span id="close"></span>
## Close a session 

### For `qiskit-ibm-runtime` 0.13 or later releases

When the session context manager is exited, the session is put into `In progress, not accepting new jobs` status.  This means that the session will finish processing all running or queued jobs until the maximum timeout value is reached.  After all jobs are completed, the session is immediately closed. This allows the
scheduler to run the next job without waiting for the session interactive timeout,
therefore reducing the average job queueing time. You cannot submit jobs to a
closed session.

``` python
with Session(service=service, backend=backend) as session:
    estimator = Estimator()
    job = estimator.run(...)
    
# The session is no longer accepting jobs but the submitted job will run to completion    
result = job.result()
```

<Admonition type="note" title="Important">
    Previously, `session.close()` **canceled** the session.  Starting with `qiskit-ibm-runtime` 0.13, `session.close()` **closes** the session. 
</Admonition>

### For Qiskit runtime releases before 0.13

> 

Reviewer: I don't think we need to mention the behavior for pre-0.13 release, since there was no way to "close" a session (based on how this doc describe as "close").

    session.close()
```

<span id="cancel"></span>
## Cancel a session 

> If a session is canceled, the session is put into `Closed` status.  Any jobs that are already running continue to run but queued jobs are put into a failed state and no further jobs can be submitted to the session. Therefore it is recommended that you cancel a session only after all session jobs **complete**, rather than immediately after they have all been submitted. 

Reviewer: ```suggestion
If a session is canceled, the session is put into `Closed` status.  Any jobs that are already running continue to run but queued jobs are put into a failed state and no further jobs can be submitted to the session. This is a convenient way to quickly fail all jobs within a session. 
```

### For Qiskit runtime releases 0.13 or later

Use the `session.cancel()` method to cancel a session.  

``` python
with Session(service=service, backend=backend) as session:

>     estimator = Estimator()
>     job = estimator.run(...)
>     # Do not cancel the session here, the job might not be completed!
>     result = job.result()
>     # Reaching this line means that the job is finished.
>     session.cancel()

Reviewer: ```suggestion
    estimator = Estimator()
    job1 = estimator.run(...)
    job2 = estimator.run(...)
    # You can use session.cancel() to fail all jobs, for example, 
    # if you realize you made a mistake.
    session.cancel()
```

    print(f">>> Expectation value from the estimator job: {estimator_job.result().values[0]}")
```

<span id="session-details"></span>
## Query session status

You can query the status of a session using `session.status()`.  You can also view a session's status on the Jobs page for your channel.

Session status can be one of the following:

> - `Pending`: Session has not started or has been deactivated. 

Reviewer: ```suggestion
- `Pending`: Session has not started or has been deactivated. The next session job needs to wait in the queue like other jobs. 
```
